mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -111,7 +111,7 @@
     {
       "run": "after",
       "snippet": "ALTER TABLE journal_records ADD COLUMN IF NOT EXISTS title text NULL;",
-      "fromModuleVersion": "mod-source-record-manager-2.5.0"
+      "fromModuleVersion": "mod-source-record-manager-2.4.0"
```

Comment:
We don't need to decrease the `fromModuleVersion`. Please, leave the version as is, it should be 2.5.0.

Reply:
@RuslanLavrov Thank you! I`ll close this one.